specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-01-02-preview/managedClusters.json

```diff
@@ -4729,6 +4739,10 @@
       "type": "string",
       "description": "The ID for Proximity Placement Group."
     },
+    "HostGroupID": {
```

Comment:
IMO, don't bother defining a specific property here - this is only used in a single place and you can instead just make above (on line 2833) be:
```
          "type": "string",
          "description": "The ID for Dedicated Host Group."
```

Which reads clearer too because you don't have to go figure out what a `HostGroupID` is.